internal/mock/mock_deviceclient.go

```diff
@@ -44,7 +43,7 @@ func (dc *DeviceClientMock) Device(id string) (models.Device, error) {
 }
 
 func (dc *DeviceClientMock) DeviceForName(name string) (models.Device, error) {
-	var device = models.Device{Id: bson.ObjectIdHex("5b977c62f37ba10e36673802"), Name: name}
+	var device = models.Device{Id: "5b977c62f37ba10e36673802", Name: name}
```

Comment:
Suggest generating a UUID as string here for consistency.

Reply:
issue #158 is opened.  After merging this PR, I will modify all the unit tests to use UUID format